client/src/components/Allocations/AllocationsModals/AllocationsTeamViewModal/AllocationsTeamViewModal.jsx

```diff
@@ -90,7 +90,7 @@ const AllocationsTeamViewModal = ({
           {isManager && <Tab label="Manage Team" />}
         </Tabs>
       </ModalHeader>
-      <ModalBody className={selectedTab === 0 ? 'd-flex p-0' : 'p-2'}>
+      <ModalBody className={selectedTab === 0 && 'd-flex p-0'}>
```

Comment:
Use the default modal margin for Manage Team modal, because it looks close to design[^1] and uses less code (here and here).

[^1]: Which is rarely pixel perfect. M.S. is not interested in standardizing spacing, so we can eyeball it. I accept this because standardizing spacing will slow down design with (I think) negligible gain. But I like standardizing anything, so definitely talk to me if you think devs would benefit.

Reply:
I think this is causing a validation error:
```
Warning: Failed prop type: Invalid prop `className` of type `boolean` supplied to `ModalBody2`, expected `string`.
```

Reply:
Thanks. Fixed validation (7fcfddd) and spacing (c82db8c).